### ISSUE TITLE
[Question]: 如何改为4个处理器的cpu版本？python -u -m paddle.distributed.launch --gpus "0" finetune.py 

### ISSUE THREAD

**Issue by Molly-Xia** [PaddlePaddle/PaddleNLP#5237]
### 请提出你的问题

你好，请问在运行python -u -m paddle.distributed.launch --gpus "0" finetune.py \
  --train_path ./data/train.json \
  --dev_path ./data/dev.json \
  --save_dir ./checkpoint \
  --learning_rate 1e-5 \
  --batch_size 16 \
  --max_seq_len 512 \
  --num_epochs 3 \
  --model uie-senta-base \
  --seed 1000 \
  --logging_steps 10 \
  --valid_steps 100 \
  --device gpu
要怎么修改关于cpu的信息？我是在服务器上运行的，服务器装了4个处理器。我在当初下载paddlepaddle时是下载的是cpu版本

**Comment by LiuChiachi** [comment]
```
python finetune.py
--train_path ./data/train.json
--dev_path ./data/dev.json
--save_dir ./checkpoint
--learning_rate 1e-5
--batch_size 16
--max_seq_len 512
--num_epochs 3
--model uie-senta-base
--seed 1000
--logging_steps 10
--valid_steps 100
--device cpu
```
